src/Paulus/Exception/Http/Standard/HttpExceptionFactory.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Paulus - A PHP micro-framework for creating RESTful services
+ *
+ * @author      Trevor Suarez (Rican7)
+ * @author      Mark Costello (mcos)
+ * @copyright   2015 Trevor Suarez
+ * @link        https://github.com/Rican7/Paulus
+ * @license     https://github.com/Rican7/Paulus/blob/master/LICENSE
+ * @version     2.0.0
+ */
+
+namespace Paulus\Exception\Http\Standard;
+
+use Exception;
+
+/**
+ * HttpExceptionFactory
+ *
+ * Provides the means to create Api Exceptions using minimal other information.
+ *
+ * @uses Exception
+ * @package Paulus\Exception\Http\Standard
+ */
+class HttpExceptionFactory
+{
+    /**
+     * Create an HTTP exception from nothing but an HTTP code
+     *
+     * @param int $code The HTTP status code
```

Comment:
The beginning of the parameter description should be indented to the "next" column after the longest line in the doc-blocks. Like this: https://github.com/mcos/Paulus/blob/0427041fd9c7eda33b9523d3d2bae9f474c35a41/src/Paulus/Paulus.php#L469-L472

Yea, I know, its dumb (I don't follow that in my newer projects), but its the current style of this repo